interactions/client/bot.py

```diff
@@ -363,6 +365,55 @@ async def change_presence(self, presence: ClientPresence) -> None:
         """
         await self._websocket._update_presence(presence)
 
+    async def get(
+        self, model: T, http_method: Optional[str] = None, **kwargs: Dict[str, Snowflake]
+    ) -> T:
+        r"""
+        A helper function for getting a model from the Discord API.
+
+        :param model: The model to get.
+        :type model: object
+        :param http_method: The HTTP method to use instead of `get_x`.
+        :type http_method: Optional[str]
+        :param \**kwargs: The arguments to pass to the HTTP method.
+        :type \**kwargs: Dict[str, Snowflake]
+        :return: The model.
+        :rtype: object
+        """
+        if model is Role:
```

Comment:
Why are we making a check for what the object actually is?

Reply:
So should I use an equality check (`==`) instead?

Reply:
We shouldn't be checking the type at all. Refer to screenshot attached https://github.com/interactions-py/library/pull/702#issuecomment-1088147432 for more details.

Reply:
There is no `get_role`, that's why `Role` is the only outlier

Reply:
My recommendation would be making a PR associated with implementing that feature

Reply:
Okay thanks.